frontend-cert/react-projects/reusable-footer/user-stories.md

```diff
@@ -0,0 +1,12 @@
+1. You should have a `div` with an `id` of `root`.
+1. You should have a `script` tag that specifies you are using `babel` to compile `JSX`.
+1. You should have a `Footer`component.
+1. Your `Footer` component should return a `footer` element.
+1. The `footer` should have:
+
+- Three unordered lists each with two list items.
+- A copyright paragraph.
+- Three icons for social media.
```

Comment:
I feel like we need to specify that they should use HTML codes or entities here.

Also, I feel like this should be rephrased to a paragraph with a copyright symbol.

Reply:
From a testing perspective maybe the elements here needs to be specified a little bit.
Because I know we want to keep things more open ended, but I am not sure how we are going to be able to test just for 3 icons. 

Looking at your implementation, it looks like the icons need to be links.
So maybe just specify that they need at least 3 links where the href value is set to `#` and the link content is set to an icon of they choice. Or something like that